src/qibolab/_core/pulses/envelope.py

```diff
@@ -254,6 +255,24 @@ def __eq__(self, other) -> bool:
         return eq(self, other)
 
 
+class Slepian(BaseEnvelope):
+    kind: Literal["slepian"] = "slepian"
+
+    relative_bandwidth: float = 0.5
+    """Relative B amplitude (wrt A)."""
```

Comment:
Not clear what `A` and `B` are.

No reference is provided, and they are not even cited in the SciPy docs of [`scipy.signal.windows.dpss`](https://scipy.github.io/devdocs/reference/generated/scipy.signal.windows.dpss.html).

Reply:
This PR is still a draft, this means it is not ready for a review. In this specific case, I started copy-pasting the SNZ and applied just the essential changes. About the motivations, at the moment it is a bit convoluted to explain, I hope it will be clearer later when @Luca-Ben-Herrmann and I will finish our investigations. At the moment this PR should be consider as a POC and probably it will not be merged.